server/odc-service/src/main/java/com/oceanbase/odc/service/notification/helper/ChannelConfigValidator.java

```diff
@@ -103,4 +116,16 @@ private void validateWebhookChannelConfig(WebhookChannelConfig channelConfig) {
 
     }
 
+    private boolean checkHostInWhiteList(String webhook) {
```

Comment:
ODC already existed an SSRF Checker, reference to `PreConditions#validInHostWhiteList`

Reply:
> seems we can use PreConditions.validInUrlWhiteList straightly. which is configured by system configuration `odc.integration.url-white-list straightly`

Should we not mix the white lists of these two modules?

Reply:
> > seems we can use PreConditions.validInUrlWhiteList straightly. which is configured by system configuration `odc.integration.url-white-list straightly`
> 
> Should we separate the white lists of these two modules?

I'm okay ,as it is security part, not business feature

Reply:
sure, I've fixed this.
Now we use the `SSRFChecker` straightly and the config `odc.integration.url-white-list `